code/modules/projectiles/guns/energy/kinetic_accelerator.dm

```diff
@@ -275,9 +275,6 @@
 		return ..()
 
 /obj/item/borg/upgrade/modkit/action(mob/living/silicon/robot/R)
-	if(..())
-		return
```

Comment:
Surely an `if(!..()) return` would be better here?

Reply:
The reason I removed it completely is that, for the moment, modkits don't need any of the procs in `/obj/item/borg/upgrade/proc/action`, since they're not real borg upgrades. I can re-add an `if(!..()) return` if you want though.

Reply:
For the sake of completeness, please do